src/plugins/merge.ts

```diff
@@ -55,11 +55,23 @@ export class Merge extends ManifestPlugin {
     }
     logger.info(`Merging ${candidates.length} pull requests`);
 
+    const [inScopeCandidates, outOfScopeCandidates] = candidates.reduce(
```

Comment:
rather than `[[], []] as CandidateReleasePullRequest`, I think you could:

```
candidtes.reduce<Array<Array<CandidateReleasePullRequest>>
```

Or something like that, and get type safety.

Reply:
Fixed, that works